module/support/logging/DataLogging/data/config/DataLogging.yaml

```diff
@@ -2,23 +2,34 @@ log_level: INFO
 
 output:
   directory: recordings
-  split_size: 5 * GiB
+  split_size: 50 * GiB
```

Comment:
Do we want to increase it by this much? The smaller chunks might be good for GDrive but also we could just change this for practice games locally.

Reply:
Definitely keep chunks smaller is better, Honestly I think 5GiB is larger than it should be

Reply:
If we keep this then, I think it would be great if we update recordings to have subfolders for every time its ran, so it's easy to figure out what set of recordings belong together